backend/typescript/models/waitlistedCamper.model.ts

```diff
@@ -8,6 +8,7 @@ export interface WaitlistedCamper extends Document {
   contactName: string;
   contactEmail: string;
   contactNumber: string;
+  status: string;
```

Comment:
should make a type `WaitlistedCamperStatus` in types.ts, import the type here and change the type of status to this
ex. `export type WaitlistedCamperStatus = "NotRegistered" | "RegistrationFormSent" | "Registered";`
```suggestion
  status: WaitlistedCamperStatus;
```